ui/src/dashboards/actions/notes.ts

```diff
@@ -100,13 +77,60 @@ export const createNoteCell = (dashboardID: string) => async (
   return dispatch(createCellWithView(dashboard, view))
 }
 
-export const updateViewNote = () => async (dispatch, getState: GetState) => {
+export interface SetNoteStateAction {
+  type: 'SET_NOTE_STATE'
+  payload: Partial<NoteEditorState>
+}
+
+export const setNoteState = (
+  noteState: Partial<NoteEditorState>
+): SetNoteStateAction => ({
+  type: 'SET_NOTE_STATE',
+  payload: noteState,
+})
+
+export const loadNote = (id: string) => async (
+  dispatch: Dispatch<Action>,
+  getState: GetState
+) => {
+  const {
+    views: {views},
+  } = getState()
+  const currentViewState = views[id]
```

Comment:
There's a [selector](https://github.com/influxdata/influxdb/blob/243f1ea5112058b74f838d0eae8da8d03f4795b7/ui/src/dashboards/selectors/index.ts#L5-L7) for this if you are so inclined